release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2913,3 +2913,43 @@ $ oc adm release info 4.13.2 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-3"]
+=== RHSA-2023:3537 - {product-title} 4.13.3 bug fix and security update
+
+Issued: 2023-06-13
+
+{product-title} release 4.13.3, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:3537[RHSA-2023:3537] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3536[RHSA-2023:3536] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.3 --pullspecs
+----
+
+[id="ocp-4-13-3-features"]
+==== Features
+
+[id="ocp-4-13-ipxe-ztp"]
+===== Support for iPXE network booting with ZTP
+{ztp-first} uses the Bare Metal Operator (BMO) to boot {op-system-first} on the target host as part of the deployment of spoke clusters. With this update, {ztp} leverages the capabilities of the BMO by adding the option of Preboot Execution Environment (iPXE) network booting for these RHCOS installations.
+
+[NOTE]
+====
+To use iPXE network booting, you must use {rh-rhacm-first} 2.8 or later.
+====
+
+For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and {ztp}].
+
+[id="ocp-4-13-3-bug-fixes"]
+==== Bug fixes
+
+* Previously on {sno}, there was a race condition present after reboot that caused pods to fail with `UnexpectedAdmissionError` error. This condition occured because The virtual distributed unit (vDU) application fails to start completely because the MAC address of the virtual functions (VFs) on the E-810C network interface controller NIC's MAC are not available when the (vDU) application starts. With this update, the resources requested by the pod are only allocated if the device plugin has registered itself to Kubelet AND healthy devices are present on the node to be allocated. (link:https://issues.redhat.com/browse/OCPBUGS-14438[*OCPBUGS-14438*]).
```

Comment:
```suggestion
* Previously on {sno}, in case of node reboot there was a race condition that could result in admission of application pods requesting devices on the node even if devices were unhealthy or unavailable to be allocate. This resulted in runtime failures when the application tried to access devices. With this update, the resources requested by the pod are only allocated if the device plugin has registered itself to Kubelet AND healthy devices are present on the node to be allocated. If these conditions are not met, the pod can fail at admission with `UnexpectedAdmissionError` error (which is expected behavior). If application pod is part of deployments (which is recommended), in case of failure subsequent pods would be spinned up which would ultimately successfully run when devices are suitable to be allocated. (link:https://issues.redhat.com/browse/OCPBUGS-14438[*OCPBUGS-14438*]).
```

Reply:
@swatisehgal we try to avoid making claims about recommendations. https://redhat-documentation.github.io/supplementary-style-guide/#recommend